include/render/drm_dumb_allocator.h

```diff
@@ -0,0 +1,32 @@
+#ifndef RENDER_DRM_DUMB_ALLOCATOR_H
+#define RENDER_DRM_DUMB_ALLOCATOR_H
+
+#include "render/allocator.h"
+
+#include <wlr/types/wlr_buffer.h>
+
+struct wlr_drm_dumb_buffer {
+	struct wlr_buffer base;
+	struct wl_list link; // wlr_drm_dumb_allocator::buffers
+
+	int drm_fd;
+
+	uint32_t id;
+	uint32_t handles[4];
```

Comment:
Dumb buffers only ever have a single plane, so we don't need arrays here.

Reply:
Is it the same for the strides and offsets?

Reply:
Yeah

Reply:
I removed `offsets` since it's always 0.